website/content/docs/hcp/get-started/connect-to-target.mdx

```diff
@@ -87,11 +88,11 @@ These are written as `boundary connect <subcommand>` and are supported for the f
 
 Refer to the [Connect helpers](/boundary/docs/concepts/connection-workflows/connect-helpers) documentation for more information.
 
-## Built-in vs. exec
+## Built-in verses exec
```

Comment:
Is it common practice to write this out like this? I've not seen it before. Isn't it `versus`? In any case I prefer `vs.`, but we might want to reword it altogether.

Reply:
I usually write this as `vs.` also, but I checked the Google style guide and they do actually recommend that it be spelled out (TIL).

I agree it might just be better to reword though; the "vs" construction was from an older version of this content from a tutorial that compared/contrasted built-in and exec, but since this section just focuses on the Exec flag I think we could just rename it as `Exec flag`.